wandb/sdk/artifacts/artifact.py

```diff
@@ -1086,53 +1074,27 @@ def __setitem__(self, name: str, item: data_types.WBValue) -> ArtifactManifestEn
             The added manifest entry
 
         Raises:
-            ArtifactFinalizedError: if the artifact has already been finalized.
-
-        Examples:
-            Basic usage:
-            ```python
-            artifact = wandb.Artifact("my_table", type="dataset")
-            table = wandb.Table(
-                columns=["a", "b", "c"], data=[(i, i * 2, 2**i) for i in range(10)]
-            )
-            artifact["my_table"] = table
-
-            wandb.log_artifact(artifact)
-            ```
-
-            Retrieving an object:
-            ```python
-            artifact = wandb.use_artifact("my_table:latest")
-            table = artifact["my_table"]
-            ```
+            ArtifactFinalizedError: If the artifact is already finalized.
```

Comment:
what does it mean for the artifact to already be finalized @ibindlish? Can't have more versions logged to it?

I've seen users get this error message and never having seen this word before / knowing what this means so want to be more specific here @ngrayluna

Reply:
An Artifact is an artifact version. So the concept of logging more versions to it doesn't apply here. Being finalized just means that its already been logged as a version so you can't add more data to the same artifact and expect the version to remain the same. You'd have to log another _version_

Reply:
Ok thanks! @ngrayluna I think for this error msg would be helpful to add another line of context (here and in all the other places it exists in this file) - `"If the artifact is already finalized. You cannot make changes to the current artifact version and instead must log a new artifact version."`